pkg/riff/commands/channel.go

```diff
@@ -18,20 +18,23 @@ package commands
 
 import (
 	"fmt"
-	"github.com/projectriff/riff/pkg/core/tasks"
 
 	"github.com/knative/eventing/pkg/apis/eventing/v1alpha1"
-	"k8s.io/api/core/v1"
-
 	"github.com/projectriff/riff/pkg/core"
+	"github.com/projectriff/riff/pkg/core/tasks"
 	"github.com/projectriff/riff/pkg/env"
 	"github.com/spf13/cobra"
+	v1 "k8s.io/api/core/v1"
 )
 
 func Channel() *cobra.Command {
 	return &cobra.Command{
 		Use:   "channel",
-		Short: "Interact with channel related resources",
+		Short: "[DEPRECATED] Interact with channel related resources",
+		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+			cmd.OutOrStderr().Write([]byte("Knative eventing is deprecated, and will be removed in a future release\n"))
```

Comment:
maybe this should just say "channel is deprecated"

also we might want to say "will be replaced"... could even mention "streams" (worst case, we change names) so that it's clear that the same general use-cases will still be supported